internal/ospath/ospath_test.go

```diff
@@ -21,14 +28,21 @@ func TestChild(t *testing.T) {
 	f.TouchFiles(paths)
 
 	f.assertChild("parent", "sibling/fileD", "")
+	fmt.Println("---")
```

Comment:
?

Reply:
oh lol this wasn't supposed to be in here